gapic-generator/templates/default/service/client/method/docs/_deprecated.erb

```diff
@@ -0,0 +1,6 @@
+<%- assert_locals method -%>
+<%- if method.is_deprecated? -%>
+#
```

Comment:
Remove the extra "blank comment line". There is already a blank comment line before the call to this subtemplate; this line adds a second.

Reply:
Done.